client/src/utils/quicknav/QuickNav.container.js

```diff
@@ -135,10 +136,21 @@ class QuickNavContainerWithRouter extends Component {
   }
 
   onSuggestionSelected(event, { suggestion, suggestionValue, suggestionIndex, sectionIndex, method }) {
+    if (this.bar.get("suggestions") === undefined)
```

Comment:
I have a preference for `== null` which also will cover `undefined`. I think it is more readable and flexible. We do not have an official coding guideline on this, but I think we should discuss it.

```suggestion
    if (this.bar.get("suggestions") == null)
```